test/unit.js

```diff
@@ -26,7 +26,62 @@ describe('bore', () => {
     expect(<Fn />.localName).to.equal('div');
   });
 
-  it('setting attributes', () => {
+  it('setting attributes on CustomElement', () => {
```

Comment:
You shouldn't need to test a custom element here. It's all just DOM. You should just be able to test a `<div />` with something like:

```js
const el = mount(<div prop={true} attributes={{ test: 'test' }} />);
expect(el.prop).to.equal(true);
expect(el.hasAttribute('prop')).to.equal(false);
expect(el.attr).to.equal(undefined);
expect(el.getAttribute('attr')).to.equal('test');
```